cluster/gce/gci/configure-helper.sh

```diff
@@ -148,6 +170,15 @@ function config-ip-firewall {
     echo "Add rule for metadata concealment"
     iptables -w -t nat -I PREROUTING -p tcp -d 169.254.169.254 --dport 80 -m comment --comment "metadata-concealment: bridge traffic to metadata server goes to metadata proxy" -j DNAT --to-destination 127.0.0.1:988
```

Comment:
Can we change this line to also use the function above?  Maybe there's a more general function for this?

Or maybe we could at least move 169.254.169.254 to a variable?  The function added is much longer than just copy/pasting the two iptables invocations, and with no other callers it doesn't seem like it's worth the extra complexity.

Reply:
I went the variable route.  There's enough differences between the commands that I don't think generalizing a function is much better than just calling iptables directly.  But the idea of the separate new function was that more similar invocations would be added in the future.